org.sonarlint.eclipse.ui/intro/RELEASE_NOTES.html

```diff
@@ -20,9 +20,28 @@ <h1>{VERSION}</h1>
 -->
 
 <div>
-	<h1>10.9.1</h1>
+	<h1>10.10</h1>
 	<p>
 		See the <a href="https://github.com/SonarSource/sonarlint-eclipse/releases">GitHub Release page</a>
+		<ul>
+			<li>Rebranding from SonarLint to SonarQube for IDE</li>
```

Comment:
I think it's worth mentioning that we fixed an issue where the disk space would grow indefinitely

Reply:
Yes, I will add this. In Eclipse, it is not globally, but per workspace, but still worth a note!